config/_default/menus.fr.yaml

```diff
@@ -21,8 +21,8 @@ footer:
   - name: Jeux de données
```

Comment:
You forgot to update the main menu in french

Reply:
well catched, thanks!